fe/fe-core/src/main/java/com/starrocks/load/LockManager.java

```diff
@@ -0,0 +1,249 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.load;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import com.google.common.collect.Sets;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.locks.ReentrantLock;
+
+// non-reentrant lock manager
+public class LockManager {
```

Comment:
Seems this class isn't general lock manager.

Reply:
How about PartitionLockManager